File: 프로그래머스/0/181837. 커피 심부름/커피 심부름.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<string> order) {
    int answer = 0;
    for(auto i : order)
    {
        if(i.find("latte") != string::npos)
            answer += 5000;
        else
            answer += 4500;
    }
    return answer;
}

File: 프로그래머스/0/181837. 커피 심부름/커피 심부름.cs
using System;

public class Solution {
    public int solution(string[] order) {
        int answer = 0;
        foreach(string menu in order)
        {
            if(menu.Contains("latte"))
                answer += 5000;
            else
                answer += 4500;
        }
        return answer;
    }
}